backend/pkg/console/console_integration_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build integration
+
+package console
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/testcontainers/testcontainers-go/modules/redpanda"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/franz-go/pkg/kgo"
+)
+
+var testSeedBroker string
+
+const TEST_TOPIC_NAME = "test.redpanda.console.topic"
```

Comment:
I'd prefer to use testify's TestSuite for the setup so that we can avoid pkg wide variables and have clearer scoping, as well as the possibility to have TearDown steps, can share certain things across containers (e.g. the testcontainer instances or clients where it makes sense).